packages/widget/src/hooks/useAutoSetAddress.ts

```diff
@@ -46,10 +45,12 @@ export const useAutoSetAddress = () => {
     route?.requiredChainAddresses?.forEach((chainID, index) => {
       res[index] = {
         chainID,
+        address: chainAddresses[index]?.address,
+        chainType: chainAddresses[index]?.chainType,
```

Comment:
this is unnecessary because these value is always undefined this will ran at initial step so should be undefined. Did you find a case that these value is not undefined?

Reply:
yes, it overrides the previously set values if this isnt specified, because this hook is called again after the 2 signature warning

Reply:
I see, nice catch